src/main/resources/static/api.yaml

```diff
@@ -855,3 +872,34 @@ components:
             attribute: true
           format: <user-id>@<ncg-id>
           example: batman@ncg
+
+    LinkedCareContexts:
+      type: object
+      properties:
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/links'
+    links:
+      type: object
+      properties:
+        hip:
+          type: object
+          properties:
+            id:
```

Comment:
mandatory/required

Reply:
can't we use the same reference in the domain model? if not, use a different object